test/functional/sync_plan/list_test.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+
+module HammerCLIKatello
+  describe SyncPlanCommand::ListCommand do
```

Comment:
You don't need to write a list_test if you're only going to touch the `info` command.